api_app/api/routes/workspaces.py

```diff
@@ -147,8 +147,10 @@ async def patch_workspace(workspace_patch: ResourcePatch, workspace=Depends(get_
     try:
         patched_workspace = workspace_repo.patch_workspace(workspace, workspace_patch, etag, resource_template_repo)
         return WorkspaceInResponse(workspace=patched_workspace)
-    except (CosmosAccessConditionFailedError):
+    except CosmosAccessConditionFailedError:
         raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail=strings.ETAG_CONFLICT)
+    except ValidationError as v:
+        raise HTTPException(status_code=400, detail=v.message)
```

Comment:
This should use the enum, i.e. `status.HTTP_400_USER_ERROR` or whatever its called in the status enum